app/App/Account/Account/Balances/BalancesExpanded/index.js

```diff
@@ -57,12 +57,13 @@ class BalancesExpanded extends React.Component {
         const isNative = isNativeCurrency(rawBalance.address)
         const nativeCurrencyInfo = networksMeta[rawBalance.chainId].nativeCurrency || {}
 
-        const rate = isNative ? nativeCurrencyInfo : rates[rawBalance.address || rawBalance.symbol] || {}
-        const logoURI = isNative ? nativeCurrencyInfo.icon : rawBalance.logoURI
-        const name = isNative ? nativeCurrencyInfo.name || networks[rawBalance.chainId].name : rawBalance.name
-        const decimals = isNative ? 18 : rawBalance.decimals
+        const rate = (isNative && nativeCurrencyInfo )|| rates[rawBalance.address || rawBalance.symbol] || {}
```

Comment:
fwiw I'm not sure this is better than ternary in these cases because the raw balance should never have any native currency data for these fields. it should just be a fine no-op though, but just test it to be sure we don't get any unexpected behavior

Reply:
Ternaries were definitely the right choice for the more complex.. the above chaining resulted in weirdness re: token names.